esrally/telemetry.py

```diff
@@ -1067,13 +1067,15 @@ def record(self):
             stats = self.client.transport.perform_request("GET", stats_api_endpoint, params={"level": level})
         except elasticsearch.NotFoundError as e:
             if "No searchable snapshots indices found" in e.info.get("error").get("reason"):
-                msg = f"Unable to find valid indices while collecting searchable snapshots stats on cluster [{self.cluster_name}]"
-                self.logger.info(msg)
+                self.logger.info(
+                    "Unable to find valid indices while collecting searchable snapshots stats "
+                    "on cluster [%s]", self.cluster_name)
                 # allow collection, indices might be mounted later on
                 return
         except elasticsearch.TransportError:
-            msg = f"A transport error occurred while collecting searchable snapshots stats on " \
-                  f"cluster [{self.cluster_name}]"
+            msg = f"A transport error occurred while collecting searchable snapshots stats on cluster " \
+                  f"[{self.cluster_name}]"
+
             self.logger.exception(msg)
             raise exceptions.RallyError(msg)
```

Comment:
As this error will be caught elsewhere, does it make sense *not* to log this in the line above? If we want to keep the log, I wonder whether we should instead [raise from None](https://stefan.sofa-rockers.org/2020/10/28/raise-from/).